### ISSUE TITLE
Serve issues in NGSI-LD format

### ISSUE THREAD

**Issue by dkastl** [gtt-project/redmine_gtt_fiware#4]
Add a link to each issue to serve it in NSGI-LD format, similar to how an issue is available under GeoJSON format for example.

**Comment by dkastl** [comment]
Repositioned by the Phase 3 design (see the [design write-up on #69](https://github.com/gtt-project/redmine_gtt_fiware/issues/69#issuecomment-5113168320)): issues reach the broker by **push** (the plugin upserts `Issue` entities), so serving issues in NGSI-LD format over HTTP becomes the deferred **pull side** - relevant when a broker should federate queries to Redmine as a context source, and only once a concrete consumer exists. The instance-published JSON-LD context (design step 2) covers the schema half of this issue.
